Library/ViewModels/ProjectPamphletMainCellViewModel.swift

```diff
@@ -255,6 +255,10 @@ public final class ProjectPamphletMainCellViewModel: ProjectPamphletMainCellView
 
         AppEnvironment.current.optimizelyClient?.track(eventName: "Campaign Details Button Clicked")
       }
+
+    self.notifyDelegateToGoToCreator.observeValues { project in
```

Comment:
Right idea but I think it would better to observeValues on the signal of `creatorButtonTappedProperty` instead.

Reply:
So the tickets needs project, creating a new Signal for `creatorButtonTappedProperty `, and `project` would not be different from `notifyDelegateToGoToCreator`. So `notifyDelegateToGoToCreator` is essentially what I need.

Reply:
I see what you mean.